src/main/cliapp/package.json

```diff
@@ -27,8 +28,7 @@
         "react-query": "^3.19.6",
         "react-router-dom": "^5.2.0",
         "react-scripts": "4.0.0",
-        "react-transition-group": "^4.4.1",
-        "http-proxy-middleware": "^2.0.1"
+        "react-transition-group": "^4.4.1"
```

Comment:
Also we try not to commit this file unless making changes to the UI.

Reply:
Sure. I don't know why this file got mixed around...